synapse/cortex.py

```diff
@@ -448,24 +448,6 @@ def getFeedFunc(self, name):
         '''
         return self.feedfuncs.get(name)
 
-    def _addSynNode(self, snap, items):
```

Comment:
I was trying to see where I could call `snap.warn` from without stubbing stuff out and realized that this method is not used.